mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -222,4 +222,14 @@ public void onSuccess(Call call, int statusCode, Headers headers, String respons
             }
         });
     }
+
+    public static String makeStringIrrecognizable(String string) {
+        return makeStringIrrecognizable(string, 3);
+    }
+
+    public static String makeStringIrrecognizable(String string, int showClearTextChars) {
```

Comment:
obfuscateString is probably a better name!?

Reply:
Yes, indeed. `showClearTextChars` also probably should be renamed to `clearTextCharCount`.